metadata.json

```diff
@@ -23,7 +23,7 @@
       "version_requirement": ">= 1.1.2 < 2.0.0"
     },
     {
-      "name": "puppet-systemd",
+      "name": "camptocamp-systemd",
```

Comment:
ah damn. Again I confused the repo name on github with the forge name of a module....